apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/main/java/gov/cms/model/dsl/codegen/plugin/GenerateSqlFromDslMojo.java

```diff
@@ -196,10 +195,6 @@ private List<ColumnBean> getPrimaryKeyColumns(RootBean root, MappingBean mapping
     // this map ensures no column is included twice
     final var columns = new LinkedHashMap<String, ColumnBean>();
     final var table = mapping.getTable();
-    for (JoinBean join : table.getPrimaryKeyJoinBeans()) {
-      final var joinColumn = mapping.mapJoinToParentColumnBean(root, join);
-      joinColumn.ifPresent(c -> columns.put(c.getColumnName(), c));
-    }
     for (ColumnBean column : mapping.getTable().getPrimaryKeyColumnBeans()) {
       columns.put(column.getColumnName(), column);
     }
```

Comment:
I believe `root` and `table` are now no longer used in this method

Reply:
Thanks. I have fixed these and removed some other unused methods.